scripts/setup/git.sh

```
@@ -143,59 +139,70 @@
-        log_error "gitコマンドが見つかりません"
+        echo "[ERROR] "gitコマンドが見つかりません""
         return 1
     fi
-    log_success "gitコマンドが使用可能です: $(git --version)"
+    echo "[SUCCESS] "gitコマンドが使用可能です: $(git --version)""
     return 0
 }
 
 # SSHキーの検証
 verify_ssh_keys() {
     if [ -f "$HOME/.ssh/id_ed25519" ]; then
```

Shellcheck (0.10.0)

[error] Couldn't find 'fi' for this 'if'.

(SC1046)